content/10.methods.md

```diff
@@ -165,7 +165,7 @@ An alternative approach is to assign arbitrary names to these groups such as via
 For each country, we computed the expected number of honorees by multiplying the proportion of authors whose affiliations were in that country with the total number of honorees.
 We then performed an enrichment analysis to examine the difference in country affiliation proportions between ISCB honorees and field-specific corresponding authors.
 We calculated each country's enrichment by dividing the observed proportion of honorees by the expected proportion of honorees.
-The variance of the log~2~ enrichment was estimated using the delta method with a small continuity correction to avoid dividing by 0 [@isbn:978-0849394447].
+The 95% confidence interval of the log~2~ enrichment was estimated using the Poisson model method [@isbn:978-0849394447].
```

Comment:
I think you should be able to use the following format to cite specific pages `[@isbn:978-0849394447, page XX-XX]`
https://pandoc.org/MANUAL.html#citations